.github/workflows/release.yml

```diff
@@ -36,6 +36,54 @@ jobs:
         uses: andymckay/cancel-action@0.2
         if: steps.release_exists.outputs.id != ''
 
+  build_windows:
+    if: github.repository_owner == 'viamrobotics'
+    needs: [prepare]
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-windows
+            platform: windows_x86_64
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.prepare.outputs.sha }}
+
+      - name: Install dependencies
+        uses: crazy-max/ghaction-chocolatey@v3
+        with:
+          args: install -y conan git
+
+      - name: Create package
+        shell: powershell
+        run: |
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+          refreshenv
+          conan profile detect
+          conan install . --output-folder=build-conan --build=missing -o "&:shared=False"
+          cmake . --preset conan-default
+          cmake --build --preset=conan-release --target ALL_BUILD install -j 8
```

Comment:
Just curious: but why direct invocation of `cmake` here rather than `conan build`?

Reply:
For reasons that weren't entirely clear, the `conan build` invocation resulted in a number of undefined symbols coming from the libraries that we were importing in the `CMakeLists.txt`. @lia-viam and I spent a short time trying to debug and then just determined that the `cmake` invocation was good enough to get releases built. Will fix in [RSDK-10666](https://viam.atlassian.net/browse/RSDK-10666).

Reply:
Maybe throw a `# TODO(RSDK-10666): ...` on there.

Reply:
done!